INSTALL.md

```diff
@@ -0,0 +1,157 @@
+
+the process takes a few minutes and ask for your wifi credentials toward the end so that the device can connect to your network.
+once it's done you should be able to access the webpage hosted by the ESP 360 Remote
+
+![image](https://github.com/nathmo/ESP-360-REMOTE/assets/15912256/29d30c93-fc50-4dd9-8d0a-e4df338b7313)
+
+as you can see it's empty. this is only the generic ESPHome configuration. follow to the next section to compile the firmware with the IR, RF and sensors feature
+
+# Standalone
+first follow this guide that will help you install the compiler for the ESPhome firmware
+[ESPHome tool desktop installation](https://esphome.io/guides/installing_esphome.html)
+once the tools are installed you can follow this guide to compile the template provided here for your device :
+[ESPHome tool desktop usage](https://esphome.io/guides/getting_started_command_line)
+you can now jump to the template section
+
+# ESPhome
+this option require docker but allow you to manage you fleet of esphome device
+
+![image](https://github.com/nathmo/ESP-360-REMOTE/assets/15912256/54c6994f-20ff-47d3-a7e6-2f1cb4a1465a)
+
+I assume you know the docker basics, here is a docker-compose.yml that you can use to run the ESPhome "fleet manager"
+```
+version: '3'
+services:
+  esphome:
+    container_name: esphome
+    image: ghcr.io/esphome/esphome
+    volumes:
+      - /path/to/esphome/config:/config
+      - /etc/localtime:/etc/localtime:ro
+    restart: always
+    privileged: true
+    network_mode: host
+    environment:
+      - USERNAME=test
+      - PASSWORD=ChangeMe
+```
+
+edit the username + password and the path to your configuration (just an empty folder is good, we will create the file from the web interface)
+
+once it's running you can can access it on http://127.0.0.1:6052/
+from there you can create a new device
+
+![image](https://github.com/nathmo/ESP-360-REMOTE/assets/15912256/c529b5d6-154f-40f2-a201-60daa0c56e23)
+
+once done, click on skip for the install and click "edit" instead from the main menu : 
+
+remplace the text with the one found in the [ESPHome Configuration](ESPHome Configuration/esp360-default-1.yaml) folder.
+as explained in the template section of this file
+once happy with your config you can compile by clicking save then come back to the main menu and click the "..." then install
+
+![image](https://github.com/nathmo/ESP-360-REMOTE/assets/15912256/d8344466-55ab-4160-86c9-c3f37f18ba89)
+
+choose manual download
+
+![image](https://github.com/nathmo/ESP-360-REMOTE/assets/15912256/e6ae26cc-6eb6-4dc1-970b-3d262d75b06b)
+
+the file will be compiled to a .bin that you will be able to download. you need to download the legacy format :
+
+![image](https://github.com/nathmo/ESP-360-REMOTE/assets/15912256/dbe9ecfa-dd81-4144-9331-9234900aaab9)
+
+
+![image](https://github.com/nathmo/ESP-360-REMOTE/assets/15912256/e6f7382d-27fc-405e-8511-8cb5cea83d2e)
+
+from there, give it a name and choose ESP32
```

Comment:
```suggestion
```

This step does not make sense in this flow, it would make sense if you were creating a new ESPHome device in the ESPHome Addon.